pepper-apis/dss-core/src/test/java/org/broadinstitute/ddp/db/dao/AnswerDaoTest.java

```diff
@@ -292,17 +294,17 @@ public void testCreateUpdateDelete_file() {
                     testData.getStudyId(), testData.getUserId(), testData.getUserId(),
                     "blob", "mime", "file", 123L);
             fileDao.markVerified(upload.getId());
-            var info = fileDao.findFileInfoByGuid(upload.getGuid()).get();
+            FileInfo info = fileDao.findFileInfoByGuid(upload.getGuid()).get();
```

Comment:
The only change on this line is from `var` to `FileInfo`. Was `var` causing problems here?